### ISSUE TITLE
#<TypeError: no implicit conversion of nil into String>

### ISSUE THREAD

**Issue by erisco** [bitrise-io/steps-xamarin-test-cloud-for-ios#19]
Error on version 1.1.0

```
#<TypeError: no implicit conversion of nil into String>
--- Stack trace: ---
p_src/step.rb:185:in `<main>'"]
```

**Comment by erisco** [comment]
I have also seen this trace:

```
#<TypeError: no implicit conversion of nil into String>
--- Stack trace: ---
["/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/analyzer.rb:192:in `include?'", "/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/analyzer.rb:192:in `block (2 levels) in build_test_commands'", "/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/analyzer.rb:190:in `each'", "/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/analyzer.rb:190:in `block in build_test_commands'", "/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/analyzer.rb:165:in `each'", "/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/analyzer.rb:165:in `build_test_commands'", "/private/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/xamarin-builder/builder.rb:53:in `build_test'", "/var/folders/lb/8n5bn9k975qgw662jpqdy7mm0000gn/T/bitrise526747593/step_src/step.rb:185:in `<main>'"]
```

**Comment by godrei** [comment]
Thanks for reporting @erisco !

Issue is related to our shared [xamarin-builder](https://github.com/bitrise-io/xamarin-builder), thats why the same issue exist in [android](https://github.com/bitrise-steplib/steps-xamarin-test-cloud-for-android/issues/12) version.

I created a PR with the [fix](https://github.com/bitrise-io/xamarin-builder/pull/26), after a review i will create a new step version, hopefully tomorrow.

**Comment by godrei** [comment]
Hi @erisco ,
should be fixed in [v1.1.1](https://github.com/bitrise-steplib/steps-xamarin-test-cloud-for-ios/releases/tag/1.1.1).
Thanks again for reporting!
